master/getting-started/docker/index.md

```diff
@@ -1,4 +1,47 @@
 ---
 title: Calico with Docker
 ---
-More information coming soon!
+
+Calico implements a Docker network plugin that can be used to provide routing and advanced network policy for Docker containers.
+
+Use the navigation bar on the left to view information on Calico for Docker,
+or continue reading for an overview of recommended guides to get started.
+
+
+## Installation
+
+#### [Requirements](installation/requirements)
+
+Information on running etcd and configuring Docker for multi-host networking.
+
+#### [Manual Install  Guide]({{site.baseurl}}/{{page.version}}/getting-started/docker/installation/manual)
```

Comment:
You've got two spaces between Install and Guide.

Ideally this is just called "Installation Guide" or "Integration Guide" to keep consistent.